buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -554,9 +554,7 @@ open class IdePlugin : Plugin<Project> {
             "-Dorg.gradle.public.api.includes=${PublicApi.includes.joinToString(":")}",
             "-Dorg.gradle.public.api.excludes=${PublicApi.excludes.joinToString(":")}",
             "-Dorg.gradle.integtest.executer=embedded",
-            "-Dorg.gradle.integtest.versions=latest",
-            "-Dorg.gradle.integtest.native.toolChains=default",
-            "-Dorg.gradle.integtest.multiversion=default",
+            "-Dorg.gradle.integtest.versions=default",
```

Comment:
Can we give this a more descriptive name? The intent in the IDE is to get some quick feedback, so perhaps a name that reflects that. It might also be worth using a name specific to the IDE, as we may choose different stuff when validating from the IDE vs in the quick CI stage, for example we might decide to test a single version of a single native toolchain when running from the IDE, but a single version of each tool chain in the quick CI stage.